tools/protoc-plugin/build.gradle

```diff
@@ -42,7 +42,8 @@ jar {
     }
     // Assemble "Fat-JAR" artifact containing all the dependencies.
     from {
-        configurations.compile.collect {
+        configurations.runtimeClasspath.collect {
+            println it
```

Comment:
Not sure why we need such a poor console output, it seems to be a debugging artifact. 

Either append it with more details to make it helpful, or remove.